spec/javascripts/tests/FilterTable_spec.js

```diff
@@ -1,33 +1,30 @@
 describe('filter table based on text field criteria', function () {
   'use strict';
 
-  function fireKeyup(el) {
-    el = el[0] || el; // Unwrap JQuery objects
-    var evObj = document.createEvent('UIEvents');
-    evObj.initUIEvent('keyup', true, true, window, 1);
-    el.dispatchEvent(evObj);
+  function visibleRows() {
+  	return $('.js-filter-rows').find('tr:visible')
+  }
```

Comment:
This is a much better place than getRows previously lived! Not sure where `$` is coming from now, though.